### DISCUSSION TITLE
How to add block mods protection

### DISCUSSION THREAD

**Discussion by galaxyandrew** [TownyAdvanced/Towny#5472] [Q&A]
I have a mod for forge for version 1.12.2 !block protection from opening the block menu does not work ! I'm trying to add a block to switch_ids: but nothing works
in config:
switch_ids:CHEST,SHULKER_BOXES,TRAPPED_CHEST,FURNACE,BLAST_FURNACE,DISPENSER,HOPPER,DROPPER,JUKEBOX,SMOKER,COMPOSTER,BELL,BARREL,BREWING_STAND,LEVER,PRESSURE_PLATES,BUTTONS,WOOD_DOORS,FENCE_GATES,TRAPDOORS,MINECARTS,LODESTONE,RESPAWN_ANCHOR,TARGET,wooden_door,spruce_door,BIRCH_WOOD,JUNGLE_DOOR,ACACIA_DOOR,DARK_OAK_DOOR,WOODEN_DOORS,IC2:TE,TARGET,

**Comment by LlmDl** [comment]
hit the container with a clay_brick in your hand and Towny will tell you what name to use in the switch_ids list.

**Reply by galaxyandrew** [reply]
Fenks
